tests/test_issue_1514/config.json

```diff
@@ -0,0 +1,326 @@
+
```

Comment:
Hmm, this seems like a lot of duplication - we probably don't want to maintain a new config file for each issue...

Reply:
See  #1573